spikeextractors/extraction_tools.py

```diff
@@ -312,6 +317,26 @@ def write_to_binary_dat_format(recording, save_path=None, file_handle=None,
         max_size = int(chunk_mb * 1e6)  # set Mb per chunk
         chunk_size = max_size // (recording.get_num_channels() * n_bytes)
 
+    if n_jobs is None:
+        n_jobs = 1
+    if n_jobs == 0:
+        n_jobs = 1
+
+    if n_jobs > 1:
+        if chunk_size is not None:
+            chunk_size /= n_jobs
+
+    if not recording.check_if_dumpable():
+        if n_jobs > 1:
+            n_jobs = 1
+            print("RecordingExtractor is not dumpable and can't be processed in parallel")
```

Comment:
I would prefer an error I think.

Reply:
No, wh should it throw an error? It can be saved to binary anyways, just not in parallel